src/SplStack/PHP81Implementation.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib\SplStack;
+
+use SplStack;
+use UnexpectedValueException;
+
+use function serialize;
+use function unserialize;
+
+/**
+ * Serializable version of SplStack
+ */
+class PHP81Implementation extends SplStack
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API